mvnw

```diff
@@ -161,7 +161,7 @@ find_maven_basedir() {
 concat_lines() {
   if [ -f "$1" ]; then
     # Remove \r in case we run on Windows within Git Bash
-    # and check out the repository with auto CRLF management
+    # and check out the mate.academy.repository with auto CRLF management
```

Comment:
make sure to avoid unintended changes, review before committing,be careful with project-wide replacements